data/discovery/field/contact.yaml

```diff
@@ -0,0 +1,6 @@
+cardinality: '1'
+datatype: text
```

Comment:
Is this `text` instead of `string` on purpose? What formatting are we expecting here?